markdown/usage/tutorials/nf_core_usage_tutorial.md

```diff
@@ -0,0 +1,296 @@
+This can be read either locally, on GitHub, or on the *nf-core* website.
+Each pipeline has its own webpage at `https://nf-co.re/<pipeline_name>` (*e.g.* [nf-co.re/rnaseq](https://nf-co.re/rnaseq)), including `Usage` documentation, `Output` documentation and `Parameter` documentation.
+
+In addition to this documentation, each pipeline comes with basic command line reference.
+This can be seen by running the pipeline with the `--help` flag, for example:
+
+```bash
+nextflow run nf-core/rnaseq --help
+```
+
+Example results of a pipeline run on full-sized test data can be browsed on the pipeline page, under the `aws` results tab.
+
+### Config profiles
+
+Nextflow can load pipeline configurations from multiple locations.
+To make it easy to apply a group of options on the command line, Nextflow uses the concept of [config profiles](https://www.nextflow.io/docs/latest/config.html#config-profiles).
+ *nf-core* pipelines load configuration in the following order:
+
+1. Pipeline: Default 'base' config
+    * Always loaded. Contains pipeline-specific parameters and "sensible defaults" for things like computational requirements
+    * Does *not* specify any method for software packaging. If nothing else is specified, Nextflow will expect all software to be available on the command line.
+2. Pipeline: Core config profiles
+    * All *nf-core* pipelines come with some generic config profiles. The most commonly used ones are for software packaging: `docker`, `singularity` and `conda`. To ensure reproducibility across different compute infrastructures, it is recommended to use containers instead of conda environments.
+    * Other core profiles are `debug` and `test`
+3. [nf-core/configs](https://github.com/nf-core/configs): Server profiles
+    * At run time, *nf-core* pipelines fetch configuration profiles from the [configs](https://github.com/nf-core/configs) remote repository. The profiles here are specific to clusters at different institutions.
+    * Because this is loaded at run time, anyone can add a profile here for their system and it will be immediately available for all *nf-core* pipelines.
+4. Personal configuration under `~/.nextflow/config`.
+5. Local config files given to Nextflow with the `-c` flag.
+6. Command line configuration.
+
+Multiple comma-separate config profiles can be specified in one go, so the following commands are perfectly valid:
+
+```bash
+nextflow run nf-core/rnaseq -profile test,docker
+nextflow run nf-core/rnaseq -profile singularity,debug
+```
+
+Note that the order in which config profiles are specified matters.
+Their priority increases from left to right.
+
+> Our tip: Be clever with multiple Nextflow configuration locations. For example, use `-profile` for your cluster configuration, `~/.nextflow/config` for your personal config such as `params.email` and a working directory `nextflow.config` file for reproducible run-specific configuration.
```

Comment:
With `nextflow.config` do you mean a custom config overwritting ressource requests or overwritting the actual nextflow.config?

Reply:
I agree that this could be confusing. Yes that means a custom config, I've edited this now to `config` only